MainRepository/Program.cs

```diff
@@ -2,6 +2,26 @@
 {
     private static void Main(string[] args)
     {
-        Console.WriteLine("Hello, World!");
+        if (int.TryParse(args[0], out int n) && n > 0 && n <= 25) Console.WriteLine($"The {n}th armstrong number is {Armstrong(n)}");

```

Comment:
If no arguments are passed, let the user know what's the issue. Program should not crash.